src/Symfony/Component/Security/Http/Attribute/IsGranted.php

```diff
@@ -24,19 +24,24 @@
 #[\Attribute(\Attribute::IS_REPEATABLE | \Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD | \Attribute::TARGET_FUNCTION)]
 final class IsGranted
 {
+    public readonly array $methods;
```

Comment:
should have `@var string[]` to document the exact type (especially as this is a public API)